Week2/10_program.js

```diff
@@ -0,0 +1,4 @@
+let myArray =["My age","30"]; // With array is possibale to store different data, within a single variable
```

Comment:
The assignment goes as follows:

> Can you store multiple types in an array

And you've made an array, that's fine! But you only store one type in it (`String`). Can you change it so that it will contain multiple `types` of values?